packages/ndla-tabs/src/Tabs.tsx

```diff
@@ -1,135 +1,132 @@
+      padding: ${spacing.nsmall};
+      position: relative;
+    }
+    [data-tab-panel]:focus-within,
+    [data-tab-trigger]:focus-within {
+      z-index: 3;
+      box-shadow: 0 0 0 2px ${colors.brand.primary};
     }
   }
+  &[data-variant='underlined'] {
+    [data-tab-list]:after {
+      border-bottom: 1px solid ${colors.brand.tertiary};
+      position: absolute;
+      bottom: 1.5px;
+      left: 0px;
+      content: '';
+      width: 100%;
+      z-index: -1;
+    }
 
-  handleSelect(index: number, last: number) {
-    this.setState({
-      index,
-    });
-
-    if (typeof this.props.onSelect === 'function') {
-      this.props.onSelect(index, last);
+    [data-tab-trigger] {
+      font-weight: ${fonts.weight.bold};
+      border-bottom: 5px solid transparent;
+      background-color: transparent;
+      color: inherit;
+      padding: ${spacing.xsmall} ${spacing.nsmall};
+      &:hover {
+        color: ${colors.brand.primary};
+      }
+      &[data-state='active'] {
+        color: ${colors.brand.primary};
+        border-bottom-color: ${colors.brand.primary};
+      }
+      &[data-disabled] {
+        color: ${colors.brand.grey};
+        cursor: not-allowed;
+      }
     }
   }
+`;
 
-  render() {
-    const { tabs, forceRenderTabPanel, modifier } = this.props;
-    const { index } = this.state;
-
-    const baseClass = modifier ? { [modifier]: modifier } : {};
-
-    return (
-      <div ref={this.tabRef}>
-        <ReactTabs
-          {...classes({ modifier })}
-          onSelect={this.handleSelect}
-          selectedIndex={this.state.index}
-          forceRenderTabPanel={forceRenderTabPanel}
-        >
-          <TabList
-            {...classes('list', {
-              ...baseClass,
-              singleLine: this.state.singleLineState,
-            })}
-          >
-            {tabs.map((tab, i) => (
-              <Tab
-                {...classes('tab', {
-                  selected: i === index,
-                  disabled: !!tab.disabled,
-                  ...baseClass,
-                })}
-                key={tab.key ? tab.key : i}
-                disabled={tab.disabled}
-                data-cy={`${tab.title}-video-tab`}
-              >
-                {tab.title}
-              </Tab>
-            ))}
-          </TabList>
-          {tabs.map((tab, i) => (
-            <TabPanel {...classes('panel', modifier)} key={tab.key ? tab.key : i}>
-              {assertIsFunction(tab.content) ? tab.content() : tab.content}
-            </TabPanel>
-          ))}
-        </ReactTabs>
-      </div>
-    );
-  }
-}
+const Tabs = ({ tabs, variant = 'underlined', defaultValue: defaultValueProp, ...rest }: Props) => {
+  const defaultValue =
+    tabs.find((tab) => tab.id === defaultValueProp && !tab.disabled)?.id ?? tabs.find((t) => t.disabled !== true)?.id;
+  return (
+    <TabsRoot defaultValue={defaultValue} data-variant={variant} {...rest}>
+      <List data-tab-list="">
```

Comment:
Skal denne være tom? Er det kun for styling?

Reply:
Jepp!